### DISCUSSION TITLE
How I can use NoSQL as database ?

### DISCUSSION THREAD

**Discussion by jigujigu** [filamentphp/filament#5598] [Help] [answered]
Hi All, 

How I can use NoSQL as my database ?

Thanks
Jigu

**Comment by pxlrbt** [comment, marked as answer]
Depends on the Database. Filament works with Laravel's Eloquent, so whatever works with that should be fine. Check for an Eloquent-Driver of your NoSQL DB of choice, e.g. MongoDB Driver: https://github.com/jenssegers/laravel-mongodb
